db/models.py

```diff
@@ -40,8 +48,95 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(
+        to="CinemaHall", on_delete=models.CASCADE, related_name="cinema_halls"
+    )
+    movie = models.ForeignKey(to="Movie", on_delete=models.CASCADE)
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to="User", on_delete=models.CASCADE)
```

Comment:
Use `settings.AUTH_USER_MODEL` to refer to the user model in definitions of other models.